packages/lib-classifier/src/store/SubjectStore/Subject/StepHistory/StepHistory.js

```diff
@@ -1,21 +1,17 @@
-import counterpart from 'counterpart'
 import cuid from 'cuid'
 import { autorun } from 'mobx'
 import { addDisposer, getRoot, tryReference, types } from 'mobx-state-tree'
 import { UndoManager } from 'mst-middlewares'
 
 import AnnotatedStep from './AnnotatedStep'
-import en from './locales/en'
-
-counterpart.registerTranslations('en', en)
 
 function beforeUnloadListener(event) {
   event.preventDefault()
-  return event.returnValue = counterpart("StepHistory.unloadWarning")
+  return event.returnValue = 'You have unsaved work. Are you sure you want to leave?'
```

Comment:
I'm confused what's happening here. You mentioned the browser is not using a string. Is this the string you were referring to?
Regardless, when I leave a classification in progress, I get this text:
![image](https://user-images.githubusercontent.com/9533007/156401334-96fdb60e-5870-4577-bbca-3ea2e9cbbf98.png)

Reply:
Correct. This string is not actually used by any browser, but assigning `event.returnValue` within `beforeUnloadListener` triggers the popup to remind users they have unsaved work while stepping through a classification task.